plugins/prow-job/commands/analyze-resource.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Analyze Kubernetes resource lifecycle in Prow job artifacts
```

Comment:
Is "Prow job artifacts" enough context? Seems like there's an unwritten assumption that the prowjob was an openshift e2e job which deployed a test cluster, and the kube resource analysis is done on a resource from the deployed test cluster? 

Asking because prowjobs alone are related to kube resources (prowjob itself, related pods) but this cmmand probably does not deal with these.